src/bcf/report/gene_specs.json

```diff
@@ -1063,11 +1063,15 @@
               "scale":{
                 "domain":[
                   "COSM",
+                  "COSN",
+                  "COSV",
```

Comment:
Rather than hardcoding the scale here, I think we can simply take the alphabetic prefixes of the IDs and let vega decide about the colors.

Reply:
Sure, we can do that. I wanted the mapping to stay consistent so that `COSM` always has the same color but now that I think about it probably isn't that important for that scale (compared to `IMPACT` for example).